### ISSUE TITLE
An ADA issue: the keyboard focus is not visible for combo boxes of a customer's address form

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#27]
[upwork.com/ab/f/contracts/20874295](https://www.upwork.com/ab/f/contracts/20874295)
[kitchencollection.com/customer/address/new/#contentarea](https://www.kitchencollection.com/customer/address/new/#contentarea)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.1.8](https://github.com/kitchencollection-com/core/releases/tag/0.1.8).

**Comment by dmitry-fedyuk** [comment]
![03](https://user-images.githubusercontent.com/83204/47501623-c3aed680-d887-11e8-975e-ff6aaf4a2f57.png)
